### ISSUE TITLE
Textures for Models

### ISSUE THREAD

**Issue by jtglvn** [jbrennan72/JACS_Lab_for_Morphology#2]
I think the most striking thing for me was the quality of the textures you all were able to get on the horse skull. Wow! Were there any lighting or angles that increased the quality of the textures on your models? Great project!

**Comment by jbrennan72** [comment]
Thanks for your question.  We did not use any external lighting to get those great textures on the horse skull model.  Believe it or not, that is an underground room without natural light, and just overhead fluorescent lighting!  Adding the color pack to the basic scanner made a huge difference in the model's texture.

**Comment by MorphoFun** [comment]
I agree, @jtglvn! The image quality was incredible, and especially without the need for external lights!
